idaes/generic_models/properties/core/examples/tests/test_ASU_PR.py

```diff
@@ -44,6 +44,11 @@
 # Get default solver for testing
 solver = get_solver()
 
+def _as_quantity(x):
```

Comment:
Seeing as this is turning up in a few places, should we be making this a general utility somewhere?

Reply:
Yes ... probably in Pyomo.  I am working on that walker, but it wasn't close enough to make yesterday's release deadline.  I figured the easiest thing was to put the temporary (inefficient) method here and then replace it once the efficient one was implemented.

Reply:
I was guessing that was the case, but wanted to confirm.